doc/projects_list.rst

```diff
@@ -4,7 +4,7 @@ Who uses Sphinx-Gallery
 
 Here is a list of projects using `sphinx-gallery`.
 
-* `Sphinx-Gallery <https://sphinx-gallery.github.io/auto_examples/index.html>`_
+* `Sphinx-Gallery <https://sphinx-gallery.github.io/stable/auto_examples/index.html>`_
```

Comment:
```
$ python -m sphinx.ext.intersphinx https://sphinx-gallery.github.io/dev/objects.inv > ~/Desktop/sg.txt
```
then inspecting this shows what the ref needs to be to refer to our own docs during the sphinx build:

```suggestion
* :ref:`Sphinx-Gallery <sphx_glr_auto_examples>`
```

Reply:
Oh thanks, I had no idea about this.